style.css

```diff
@@ -84,6 +108,45 @@ a:hover {
   background-color: #344563;
 }
 
+.mobile_nav_items {
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100vh;
+  display: none;
+  position: fixed;
+  flex-direction: column;
+  backdrop-filter: blur(7px);
+  padding: 70px 15px 15px 10px;
+  -webkit-backdrop-filter: blur(7px);
+  background: rgba(46, 64, 228, 0.7);
+  box-shadow: 0 8px 32px 0 rgba(31, 38, 135, 0.37);
+}
+
+.mobile_nav_items .nav_item_action {
+  width: 100%;
+  display: block;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.toggle-cross {
```

Comment:
- [Optional] We are asked to  kindly follow the design from the proposed templates in Figma(margins, fonts)
![Screenshot from 2022-11-14 11-13-27](https://user-images.githubusercontent.com/63087491/201634261-16ff6bc3-4a0c-4680-babe-59543b30f374.png)